projects/Gancio/default.nix

```diff
@@ -0,0 +1,24 @@
+{
+  pkgs,
+  lib,
+  sources,
+}@args:
+{
+  metadata = {
+    summary = "Gancio Shared agenda for local communities that supports Activity";
+      subgrants = [
+        "gancio"
```

Comment:
https://nlnet.nl/project/gancio/ returns 404 

@eljamm we need a bit of documentation in the data type with the expectations... I thought we had it but apparently we don't.

Reply:
This just needs to be uppercase: `Gancio`